Still there's an error on Darwin:

```
       > /nix/store/bp2yamylj44byk4flc9czdsf91y42pyv-coreutils-9.5/bin/install -c target/x86_64-apple-darwin/release/librusile.so rusile.so
       > install: cannot stat 'target/x86_64-apple-darwin/release/librusile.so': No such file or directory
       > make[2]: *** [Makefile:1953: rusile.so] Error 1
       > make[2]: Leaving directory '/private/tmp/nix-build-sile-0.15.8.drv-0/sile-0.15.8'
       > make[1]: *** [Makefile:1189: all-recursive] Error 1
       > make[1]: Leaving directory '/private/tmp/nix-build-sile-0.15.8.drv-0/sile-0.15.8'
       > make: *** [Makefile:773: all] Error 2
       For full logs, run 'nix log /nix/store/77ndplr9vr8wwyw4x36jc46rxflak2mg-sile-0.15.8.drv'.
```

--- a/pkgs/by-name/si/sile/package.nix
+++ b/pkgs/by-name/si/sile/package.nix
@@ -29,17 +29,17 @@
 
 stdenv.mkDerivation (finalAttrs: {
   pname = "sile";
-  version = "0.15.7";
+  version = "0.15.8";
 
   src = fetchurl {
     url = "https://github.com/sile-typesetter/sile/releases/download/v${finalAttrs.version}/sile-${finalAttrs.version}.tar.zst";
-    hash = "sha256-PjU6Qfn+FTL3vt66mkIAn/uXWMPPlH8iK6B264ekIis=";
+    hash = "sha256-ZMF6uv1bHvMEGagbAAmYhwwbCBtjctVbwx35w7g/D2o=";
   };
 
   cargoDeps = rustPlatform.fetchCargoTarball {
     inherit (finalAttrs) pname version src;
     nativeBuildInputs = [ zstd ];
-    hash = "sha256-iPkXEUC4U1m/ComIDo/J5kwkmM1QdowioNtnSnmMhJ0=";
+    hash = "sha256-xv411fxOkjsbVNCNEIaopjLHbUCWdw+1JWePXHdrKBc=";
   };
 
   nativeBuildInputs = [

